Content.Server/Zombies/ZombifyOnDeathSystem.cs

```diff
@@ -208,6 +208,7 @@ public void ZombifyEntity(EntityUid target)
             RaiseLocalEvent(new EntityZombifiedEvent(target));
             //zombies get slowdown once they convert
             _movementSpeedModifier.RefreshMovementSpeedModifiers(target);
+            Dirty(target);
```

Comment:
Does this really need to be a full entity dirty? Much more efficient to just get the comps you touched.